frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/header/custom_cover_picker.dart

```diff
@@ -322,7 +322,7 @@ class _CoverImagePreviewWidgetState extends State<CoverImagePreviewWidget> {
                     (l) => _buildImageDeleteButton(context),
                     (r) => Container(),
                   )
-                : Container()
+                : Container(),
```

Comment:
```suggestion
                : const SizedBox.shrink(),
```

Reply:
I will add a linter check and replace all the `Container()` with `SizedBox.shrink()`;